README.md

```diff
@@ -17,6 +17,15 @@ A web application for providing access to Digital Objects in SDR Storage.
   ./bin/test.sh
   ```
 
+### Note for osx users
+the current bundle requires eventmachine 1.0.7. This won't build on a modern mac
+without specifying where the ssl libraries are:
+
+```
+gem install eventmachine -v '1.0.7' -- --with-cppflags=-I/usr/local/opt/openssl/include
+```
+
```

Comment:
Thanks @jcoyne for digging into this install problem; I agree that having this comment in the README is a great place to publish this information for the team.

It looks like there are plenty of other dependencies that need updating, so I'm not sure explicitly calling this one out is particularly helpful.